javascript/javascript1/week3/smart-phone.js

```diff
@@ -0,0 +1,65 @@
+const activities = [];
+const date = new Date().toLocaleDateString();
+
+// Function for adding activity into array
+function addActivity(activity, duration) {
+    if (typeof date === "string" && typeof activity === "string" && typeof duration === "number") {
```

Comment:
overall there's not much sense of checking if date is string: we know that because it's const variable defined 2 lines above.